### ISSUE TITLE
Request for Guidance on Using UnSAM+ Training Data Merging Code

### ISSUE THREAD

**Issue by haokunchen0** [frank-xwang/UnSAM#11]
Dear Franks,

Thank you for this great work! 

I was wondering if there is any code in the repository that merges the ground-truth mask of SA-1B with your IoU-based unsupervised segmentation mask, **as described in your paper for the training data of UnSAM**+. If there is, could you please let me know how I should use it?

Thank you! I look forward to getting your help!

Kind regards,

Haokun Chen

**Comment by wenfang361** [comment]
Hi，Have you made any progress? Has the issue been resolved? I've encountered a similar problem.Plz.
